pkgs/tools/security/creddump/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitLab, python2, python2Packages }:
+
+python2Packages.buildPythonApplication rec {
+  pname = "creddump";
+  version = "0.3";
+
+  src = fetchFromGitLab {
+    owner = "kalilinux";
+    repo = "packages/creddump";
+    # url-encoding workaround: https://github.com/NixOS/nixpkgs/issues/65796#issuecomment-517829019
+    rev = "debian%2F${version}-1kali2"; # %2F = urlquote("/")
+    sha256 = "0r3rs2hggsvv619l3fh3c0jli6d3ryyj30ni3hz0nz670z5smzcf";
+  };
+
+  # No setup.py is available
+  dontBuild = true;
+  doCheck = false;
+  pythonPath = with python2Packages; [ pycrypto ];
```

Comment:
This is not used anywhere..

Reply:
This is a run-time dependency. If this is not used, you get the following error when executing one of the tools:

```
$ pwdump 
Traceback (most recent call last):
  File "/nix/store/mxbxljjdf4h05975bp594l805xcj767f-creddump-0.3/bin/.pwdump-wrapped", line 26, in <module>
    from framework.win32.hashdump import dump_file_hashes
  File "/nix/store/mxbxljjdf4h05975bp594l805xcj767f-creddump-0.3/bin/framework/win32/hashdump.py", line 24, in <module>
    from Crypto.Hash import MD5
ImportError: No module named Crypto.Hash
```

Reply:
If it's just a regular dependency, usually python packages use `propagatedBuildInputs`.